main.tf

```
@@ -594,3 +604,176 @@ resource "aws_scheduler_schedule" "this" {
     }
   }
 }
+
+resource "aws_pipes_pipe" "this" {
+  for_each = { for k, v in local.eventbridge_pipes : v.name => v if var.create && var.create_pipes }
+
+  name        = each.value.Name
```

this would need to be 
```suggestion
  name        = lookup(each.value, "name", null)
```

If you are going to support `name_prefix` on the next line. Also, I see here that `Name` is capitalized - is that intentional?

---

Dropped the support for name_prefix. Maybe I will add it in future releases.

---

Name != name. It is what it is, currently.